bindings/experimental/distrdf/python/DistRDF/HeadNode.py

```diff
@@ -377,6 +377,11 @@ def __init__(self, backend: BaseBackend, npartitions: Optional[int], localdf: RO
         if isinstance(args[0], ROOT.TTree):
             # RDataFrame(tree, defaultBranches = {})
             self.tree = args[0]
+            # TTreeIndex is not supported in distributed RDataFrame
+            list_of_friends = self.tree.GetListOfFriends()
+            if list_of_friends and list_of_friends.GetEntries() > 0:
+                if any(friend_element.GetTree().GetTreeIndex() for friend_element in list_of_friends):
+                    raise ValueError("At least one TTreeIndex was detected. This is not supported in distributed mode.")
```

Comment:
"Friend TTree '{treename}' has a TTreeIndex. This is not supported in distributed mode."